android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java

```diff
@@ -678,6 +679,15 @@ private WritableMap getImage(final Activity activity, String path) throws Except
             throw new Exception("Cannot select remote files");
         }
         BitmapFactory.Options original = validateImage(path);
+        ExifInterface originalExif = new ExifInterface(path);
```

Comment:
it's fine to read just the original exif because the orientation tag gets carried forward in the compression step